### PR TITLE
[ru] Localize ebpf.md

### PR DESCRIPTION
**Pull request by lobkovilya**
### Describe your changes

This PR adds RU localization of **eBPF**

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/ebpf.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Как правило, приложения работают в пользовательском пространстве,
+и если возникает необходимость в привилегированном доступе (например, доступ к аппаратному обеспечению),
+приложение отправляет запрос к ядру также известный как "системный вызов".
+В большинстве случает такой подход прекрасно работает. 
+Однако бывают ситуации, когда разработчики нуждаются в более низкоуровневом доступе к системе.
+Такие ситуации, как правило, связаны с наблюдаемостью (observability), безопасностью и работой с сетью.
+Использование модулей ядра Линукс позволяет решить вышеупомянутую проблему без изменения исходного кода ядра.
+Но несмотря на преимущества, такой подход влечет за собой риски безопасности.
+Код модулей ядра работает в пространстве ядра, а это значит, что сбой модуля неминуемо приводит к сбою всего ядра и как следствие — сбою всей аппаратной системы.
+Вдобавок, модули ядра имеют повышенные привилегии и прямой доступ к системным ресурсам. Без должной безопасности, они могут стать источником уязвимостей для хакеров.
```
Comment (kbatalin) [inline content/ru/ebpf.md:33]:
```suggestion
Вдобавок, модули ядра имеют повышенные привилегии и прямой доступ к системным ресурсам. Без должной безопасности, они могут стать источником уязвимостей для злоумышленников.
```

File: content/ru/ebpf.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: eBPF
+status: Completed
+category: Technology
+tags: ["architecture", "networking", "security"]
+---
+
+eBPF (англ. extended Berkley Packet Filter — расширенный фильтр пакетов Беркли) это технология,
+которая позволяет небольшим программам или скриптам запускаться в пространстве ядра ОС Линукс
```
Comment (kirkonru) [inline content/ru/ebpf.md:9]:
```suggestion
Технология eBPF (англ. extended Berkley Packet Filter — расширенный фильтр пакетов Беркли) 
позволяет небольшим программам или скриптам запускаться в пространстве ядра ОС Линукс 
```

File: content/ru/ebpf.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+status: Completed
+category: Technology
+tags: ["architecture", "networking", "security"]
+---
+
+eBPF (англ. extended Berkley Packet Filter — расширенный фильтр пакетов Беркли) это технология,
+которая позволяет небольшим программам или скриптам запускаться в пространстве ядра ОС Линукс 
+без необходимости изменять код ядра или загружать модули ядра.
+
+ОС Линукс имеет два пространства: пространство ядра (kernel space) и пользовательское (user space).
```
Comment (kirkonru) [inline content/ru/ebpf.md:12]:
```suggestion
В ОС Линукс есть два пространства: пространство ядра (kernel space) и пространство пользователя (user space). 
```

File: content/ru/ebpf.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+которая позволяет небольшим программам или скриптам запускаться в пространстве ядра ОС Линукс 
+без необходимости изменять код ядра или загружать модули ядра.
+
+ОС Линукс имеет два пространства: пространство ядра (kernel space) и пользовательское (user space). 
+Ядро представляет основу операционной системы и имеет неограниченный доступ к аппаратному обеспечению.
+
+Приложения находятся в пользовательском пространстве. 
+Когда возникает необходимость в привилегированном доступе, приложения отправляют запрос к ядру.
+Для приложений, требующих большей свободы действий, например, для прямого доступа к аппаратному обеспечению, 
+ядро может быть расширено за счёт моделей ядра Линукс.
```
Comment (kirkonru) [inline content/ru/ebpf.md:18]:
```suggestion
Для приложений, требующих большей свободы действий, например, прямого доступа к аппаратному обеспечению, 
ядро может быть расширено за счёт модулей ядра Линукс. 
```

File: content/ru/ebpf.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+без необходимости изменять код ядра или загружать модули ядра.
+
+ОС Линукс имеет два пространства: пространство ядра (kernel space) и пользовательское (user space). 
+Ядро представляет основу операционной системы и имеет неограниченный доступ к аппаратному обеспечению.
+
+Приложения находятся в пользовательском пространстве. 
+Когда возникает необходимость в привилегированном доступе, приложения отправляют запрос к ядру.
+Для приложений, требующих большей свободы действий, например, для прямого доступа к аппаратному обеспечению, 
+ядро может быть расширено за счёт моделей ядра Линукс. 
+Такой подход расширяет стандартные функции ядра, давая приложениям доступ к низкоуровневым компонентам.
```
Comment (kirkonru) [inline content/ru/ebpf.md:19]:
```suggestion
Такой подход расширяет стандартные функции ядра, открывая приложениям доступ к низкоуровневым компонентам.
```

File: content/ru/ebpf.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+Однако расширение ядра за счет модулей влечет за собой риски безопасности, делая eBPF привлекательной альтернативой.
+
+## Какую проблему решает
+
+Как правило, приложения работают в пользовательском пространстве,
+и если возникает необходимость в привилегированном доступе (например, доступ к аппаратному обеспечению),
+приложение отправляет запрос к ядру также известный как "системный вызов".
```
Comment (kirkonru) [inline content/ru/ebpf.md:26]:
```suggestion
Как правило, приложения работают в пространстве пользователя.
Если возникает необходимость в привилегированном доступе (например, к аппаратному обеспечению),
приложение отправляет запрос к ядру, также известный как «системный вызов».
```

File: content/ru/ebpf.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+eBPF (англ. extended Berkley Packet Filter — расширенный фильтр пакетов Беркли) это технология,
+которая позволяет небольшим программам или скриптам запускаться в пространстве ядра ОС Линукс 
+без необходимости изменять код ядра или загружать модули ядра.
+
+ОС Линукс имеет два пространства: пространство ядра (kernel space) и пользовательское (user space). 
+Ядро представляет основу операционной системы и имеет неограниченный доступ к аппаратному обеспечению.
+
+Приложения находятся в пользовательском пространстве.
```
Comment (kirkonru) [inline content/ru/ebpf.md:15]:
```suggestion
Приложения находятся в пространстве пользователя. 
```

File: content/ru/ebpf.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+Как правило, приложения работают в пользовательском пространстве,
+и если возникает необходимость в привилегированном доступе (например, доступ к аппаратному обеспечению),
+приложение отправляет запрос к ядру также известный как "системный вызов".
+В большинстве случает такой подход прекрасно работает. 
+Однако бывают ситуации, когда разработчики нуждаются в более низкоуровневом доступе к системе.
+Такие ситуации, как правило, связаны с наблюдаемостью (observability), безопасностью и работой с сетью.
+Использование модулей ядра Линукс позволяет решить вышеупомянутую проблему без изменения исходного кода ядра.
+Но несмотря на преимущества, такой подход влечет за собой риски безопасности.
+Код модулей ядра работает в пространстве ядра, а это значит, что сбой модуля неминуемо приводит к сбою всего ядра и как следствие — сбою всей аппаратной системы.
+Вдобавок, модули ядра имеют повышенные привилегии и прямой доступ к системным ресурсам. Без должной безопасности, они могут стать источником уязвимостей для злоумышленников.
```
Comment (kirkonru) [inline content/ru/ebpf.md:33]:
```suggestion
Увы, у этого подхода есть не только плюсы, но и минусы (в первую очередь связанные с безопасностью).
Модули ядра работают в пространстве ядра. Это означает, что сбой модуля неминуемо приводит к сбою ядра и, как следствие, падению всей системы.
Кроме того, модули ядра наделены повышенными привилегиями и имеют прямой доступ к системным ресурсам. Без должной защиты этим могут воспользоваться злоумышленники.
```

File: content/ru/ebpf.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+## Как именно решает проблему
+
+По сравнению с модулями ядра, eBPF предоставляет более контролируемую и ограниченную среду для исполнения пользовательских программ.
+eBPF программа запускается в "песочнице" внутри ядра, обеспечивая тем самым изоляцию и снижая риски.
+Если в eBPF программе обнаружится уязвимость или недочет, то их воздействие будет ограничено "песочницей".
```
Comment (kirkonru) [inline content/ru/ebpf.md:39]:
```suggestion
По сравнению с модулями ядра, eBPF позволяет запускать пользовательские программы в более контролируемой и ограниченной среде.
eBPF-программа работает в «песочнице» внутри ядра. Тем самым обеспечивается необходимая изоляция и снижаются риски.
Если в eBPF-программе обнаружится уязвимость или недочет, то их воздействие будет ограничено «песочницей».
```

File: content/ru/ebpf.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+Более того, прежде чем программа сможет запуститься внутри ядра, она должна пройти процесс верификации.
+Верификатор проверяет eBPF программу на наличие потенциально небезопасного кода,
```
Comment (kirkonru) [inline content/ru/ebpf.md:41]:
```suggestion
Верификатор проверяет eBPF-программу на наличие потенциально небезопасного кода, 
```

File: content/ru/ebpf.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+## Как именно решает проблему
+
+По сравнению с модулями ядра, eBPF предоставляет более контролируемую и ограниченную среду для исполнения пользовательских программ.
+eBPF программа запускается в "песочнице" внутри ядра, обеспечивая тем самым изоляцию и снижая риски.
+Если в eBPF программе обнаружится уязвимость или недочет, то их воздействие будет ограничено "песочницей".
+Более того, прежде чем программа сможет запуститься внутри ядра, она должна пройти процесс верификации.
+Верификатор проверяет eBPF программу на наличие потенциально небезопасного кода, 
+такого как доступ к памяти за пределами выделенного буфера, бесконечные циклы и неавторизованный доступ к функциям ядра.
+Верификация гарантирует что eBPF программа не войдет в бесконечный цикл и не приведет к сбою ядра.
+Эти меры безопасности делают eBPF более надежным вариантом запуска приложений в ядре, по сравнению с модулями ядра Линукс.
```
Comment (kirkonru) [inline content/ru/ebpf.md:44]:
```suggestion
Верификация гарантирует, что eBPF-программа не войдет в бесконечный цикл и не приведет к сбою ядра.
Такие меры безопасности делают eBPF более надежным вариантом запуска приложений в ядре по сравнению с модулями ядра Линукс.
```

**Review by kirkonru — APPROVED** [review]
Hi @lobkovilya! Thanks for the contribution! I've made a couple of edits; most of them are stylistic, meant to make it easier to read.

File: content/ru/ebpf.md (hunk 11 above)
Comment (kirkonru) [reply content/ru/ebpf.md:44]: By the way, feel free to join our [Russian Cloud Native](https://t.me/ru_cloudnative) localizer community. 😊

**Review by tym83 — APPROVED** [review]

**Pull request merged**